app/modules/screen-sharing/index.html

```diff
@@ -0,0 +1,144 @@
+                <p>Please allow access to your screen in order to begin sharing.</p>
+                <nav class="popup__wrapper__options">
+                    <button class="popup__wrapper__options__btn">Got it</button>
+                    <% if (!initiator) { %>
+                        <button class="popup__wrapper__options__btn--error">Reject</button>
+                    <% } %>
+                </nav>
+            </div>
+        </div>
+    </script>
+
+    <script type="text/tmpl" id="call-preview">
+        <div class="popup">
+            <div class="popup__wrapper">
+                <h2 class="popup__wrapper__title">Confirm your screen share.</h2>
+                <div id="video-preview" class="video-preview-pane"></div>
+                <nav class="popup__wrapper__options">
+                    <button class="popup__wrapper__options__btn">Cancel</button>
+                    <button class="popup__wrapper__options__btn--success">Start Screenshare</button>
+                </nav>
+            </div>
+        </div>
+    </script>
+
+    <script type="text/tmpl" id="user-authentication">
+        <div class="authentication">
+        <h1>Screen Sharing</h1>
+            <p>
+            The example below shows a simple screen sharing application. To test it out, input a username and hit "send" to sign in. You will then be able to see any other users who are currently signed in and establish a dual screen sharing session with them. To start sharing, simply click the name of the user you want to call in the list on the left. If you're working solo, you can open this page in multiples tabs/windows and sign in with different usernames to see the full functionality in action.
+            </p>
+            <p>
+            This example illustrates the use of the <a href="https://docs.respoke.io/js-library/respoke.Endpoint.html#startScreenShare" target="_blank">startScreenShare</a> method of the <a href="https://docs.respoke.io/js-library/respoke.Endpoint.html" target="_blank">Endpoint</a> object. This method returns a <a href="https://docs.respoke.io/js-library/respoke.Call.html" target="_blank">Call</a> object which acts as your interface into the WebRTC call and gives you events to notify you about call actions such as when audio is muted or the connection is terminated.
+            </p> 
+            <hr/>
+
+            <p class="cbl-instructions">
+                Enter your name to see a list of all others on this page right now.
+            </p>
+            <form class="cbl-name">
+                <input class="cbl-name__text" type="text" placeholder="Enter a name" />
+                <input class="cbl-name__submit" type="submit" value="Send" />
+            </form>
+        </div>
+    </script>
+
+    <script type="text/tmpl" id="video-container">
+        <div class="video-contain">
+            <div class="video-buttons">
+                <button class="video-buttons__button--cancel" id="hangup">Hangup</button>
```

Comment:
Do you have a suggestion on how to reword this? It's still a call, but we could certainly say "stop sharing" or "stop screenshare" instead.